src/Tribe/Views/V2/Admin/Bar.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Handles Admin Bar management for Views V2.
+ *
+ * @since   5.0.0
+ *
+ * @package Tribe\Events\Views\V2\Admin
+ */
+
+namespace Tribe\Events\Views\V2\Admin;
+
+use Tribe__Utils__Array as Arr;
+use WP_Admin_Bar;
+
+/**
+ * Class Bar
+ *
+ * @since   5.0.0
+ *
+ * @package Tribe\Events\Views\V2\Admin
+ */
+class Bar {
+	CONST SUSPEND_CACHE_KEY = 'tribe_events_suspend_view_html_cache';
+	CONST SUSPEND_NONCE_KEY = 'tribe_events_views_v2_suspend_view_html_cache';
```

Comment:
* :no_entry_sign: PHP keywords must be lowercase; expected "const" but found "CONST"
* :no_entry_sign: CONST keyword must be lowercase; expected "const" but found "CONST"

<sub>hash: 435cbfd526d829082f83521957b14805</sub>